### PR TITLE
Renamed createFrom to convertFrom in RasterYuv422Semipacked and made it static

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes https://github.com/cogneco/ooc-kean/issues/772

### CODE REVIEW

File: source/draw/RasterYuv422Semipacked.ooc [hunk 1]
```diff
@@ -129,6 +100,30 @@ RasterYuv422Semipacked: class extends RasterPacked {
 		fileWriter write(this buffer pointer as Char*, this buffer size)
 		fileWriter close()
 	}
+	convertFrom: static func (original: RasterImage) -> This {
+		result := This new(original size)
```
Comment (marcusnaslund) [inline source/draw/RasterYuv422Semipacked.ooc]: The other `convertFrom` check
```ooc
if (original instanceOf?(This))
	result = (original as This) copy()
```

Comment (sebastianbaginski) [inline source/draw/RasterYuv422Semipacked.ooc]: That's true, thanks.

Comment (sebastianbaginski) [inline source/draw/RasterYuv422Semipacked.ooc]: Corrected.

**Comment by marcusnaslund** [comment]
Can you create an issue for creating a test for this class?

**Comment by sebastianbaginski** [comment]
Sure: https://github.com/cogneco/ooc-kean/issues/808

**Comment by marcusnaslund** [comment]
Thanks. Looks good otherwise. If there's something wrong, we'll know with the test.

**Pull request merged**